ipatests/prci_definitions/gating.yaml

```diff
@@ -15,7 +15,7 @@ topologies:
 jobs:
   fedora-27/build:
     requires: []
-    priority: 100
+    priority: 150
```

Comment:
Did you mean to bump this to 150?

Reply:
Yes, gating uses higher priorities than nightlies (150 for build and 100 for tests in gating.yaml, 100 for build and 50 for tests in the other test definitions).